### ISSUE TITLE
[Tokens] From that page, the list of transfers for a specific token (tab in Token Detail page) should be easily discoverable.

### ISSUE THREAD

**Issue by 1Mateus** [hack-a-chain-software/indexer-kadena#62]
## EN 
- [ ] We need to develop a page that indexes all tokens listed in the contracts table. For each indexed token, a separate page should be generated, displaying a comprehensive list of all transactions associated with that specific token.
- [ ] Alternatively, we can implement a filter on the "Token Transfers" page to enable filtering by specific tokens. This is a more straightforward approach.
## PT 
- [ ]   precisamos criar uma página que indexe os tokens que vem da tabela de contracts e criar uma página para cada token indicando as transações realizadas com eles.
 - [ ] Outra maneira de fazer isso, é criar um filtro dentro da página de “Token Transfers” para poder filtrar por token. Bora fazer do jeito fácil. 

https://www.notion.so/Mat-Reply-Hackchain-Indexer-Explorer-Frontend-review-fa7e71e82bdf44cd9fc99821b484fb88?pvs=4#b8428e3316354ef198a1b65c56b618ae

**Comment by 1Mateus** [comment]
@rafaeldkb como ficou o encaminhamento disso aqui? tem ideia de como vai ser pra prosseguir com o filtro? design?

**Comment by 1Mateus** [comment]
@davibauer @rafaeldkb como falamos na daily, não deu tempo de comentarmos juntos sobre essa página.

O @rafaeldkb deu a ideia de criar um filtro e poder filtrar as transfers pelo modulo e gerar as query com o filtro pelo token.

@davibauer seria possível ter uma lista de tokens únicos? isso é na tabela de contracts, né?

**Comment by rafaeldkb** [comment]
@1Mateus não seria possível, quando não tem o token na coingecko, direcionar pra mesma página de token details, mas não mostrar os details da coingecko, apenas as transfers que puxamos da blockchain diretamente? Talvez deixar alguma mensagem tipo "essa informação não está disponível" nos campos que viriam da coingecko.

**Comment by rafaeldkb** [comment]
Tá show!

**Comment by rafaeldkb** [comment]
@1Mateus I was wondering if it would be possible of, when the token has no details, to go directly to the transfers page when someone click on the token

**Comment by bernardonery** [comment]
> [@1Mateus](https://github.com/1Mateus) I was wondering if it would be possible of, when the token has no details, to go directly to the transfers page when someone click on the token

Are you suggesting that the same thing should not happen as is happening with this token here?

![Image](https://github.com/user-attachments/assets/05627a67-793a-41e9-afab-687434190ce1)

**Comment by rafaeldkb** [comment]
Yes, I'm suggesting that when there's no token details, we send directly to the Transfers tab (which is inside the token details), instead of sending to the "Info" tab, so that we display information. 

<img width="1346" alt="Screenshot 2024-09-10 at 14 41 59" src="https://github.com/user-attachments/assets/af7db8de-fc5c-466f-b079-4119ca050614">

**Comment by rafaeldkb** [comment]
Last comment from Mateus on this:
"Essa aqui eu não consegui um exemplo, não consegui paginar a tabela e achar um token que não existe, pq ficava retornando 503 do backend."
@1Mateus pode usar o token Crankk pra testar? https://www.kadscan.io/tokens/free.crankk01?tab=info
